package-lock.json

```diff
@@ -518,6 +518,29 @@
                 "node": ">=12.0.0"
             }
         },
+        "node_modules/@microsoft/vscode-azext-azureutils/node_modules/@microsoft/vscode-azext-utils": {
```

Comment:
Technically a semver violation, but I think we should just make this `0.2.6` of `@microsoft/vscode-azext-utils`. Otherwise I have to update `@microsoft/vscode-azext-azureutils` and `@microsoft/vscode-azext-azureappservice`.